scripts/ci/credscan/CredScanSuppressions.json

```diff
@@ -111,6 +111,12 @@
         "src\\diskpool\\README.md"
       ],
       "_justification": "Faked password"
-    }
```

Comment:
Just saw this & hoping no real secrets are exposed here. If so I would recommend removing these tests completely. Thanks!